packages/features/eventtypes/components/tabs/setup/EventSetupTab.tsx

```diff
@@ -95,6 +97,21 @@ export const EventSetupTab = (props: EventSetupTabProps & { urlPrefix: string; h
               </>
             )}
           </div>
+          <div className="[&_label]:my-1 [&_label]:font-normal">
+            <SettingsToggle
+              title={t("translate_description_button")}
+              checked={!!autoTranslateDescriptionEnabled}
+              onCheckedChange={(value) => {
+                formMethods.setValue("autoTranslateDescriptionEnabled", value, { shouldDirty: true });
+              }}
+              disabled={meQuery.isPending || !meQuery.data?.organizationId}
+              tooltip={
+                !meQuery.isPending && !meQuery.data?.organizationId
+                  ? t("orgs_upgrade_to_enable_feature")
+                  : undefined
```

Comment:
NIT: Lets pull this from session rather than useMeQuery - it fetches a pretty large amount of data for what we are using it for here